REFERENCE.md

```diff
@@ -819,6 +820,14 @@ Data type: `Optional[String[1]]`
 
 
 
+Default value: ``undef``
+
+##### <a name="replicaof"></a>`replicaof`
+
+Data type: `Optional[String[1]]`
+
+
+
```

Comment:
Description is missing. Either do not update REFERENCE.md in the PR (we update this file when releasing a new version) or regenerate it and update the PR.

Reply:
Hi, can this PR be approved soon? 
"slaveof" has been deprecated for a while now...

Reply:
As @smortex said, REFERENCE.md needs to be regenerated.